http_handlers.go

```diff
@@ -78,7 +78,8 @@ type trainBulkArgs struct {
 // PackDownloadRequestArgs is the args to download pack from upstream
 type PackDownloadRequestArgs struct {
 	LangCode              string `json:"lang"`
-	PackVersionIdentifier string `json:"pack"`
```

Comment:
Naming can be simplified.

`PackDownloadRequestArgs` to something like `packDownload` (no need to export)
`PackVersionIdentifier`to `Package` 
`PackVersionIdentifier` to `Version`

Reply:
Version might get confused with the number version, identifier is a string, to avoid the confusion and be explicit, I used it. Package also means a different thing, "Pack" suits better. Perhaps using ID would be better

Reply:
`Version` does the job. no need to assume its datatype